docs/new-idp-requirements.md

```diff
@@ -0,0 +1,26 @@
+# New IDP Requirements
+
+## Summary
+
+This document describes the minimum requirements for adding a new IDP (Identity Provider) to the Sigstore Public Good Deployment.
+
+## Requirements
+
+The key words "MUST", "MUST NOT", "SHOULD", "SHOULD NOT", and "MAY" in this document are
+to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+A new IDP must meet the following requirements:
+
+- MUST host a `/.well-known/openid-configuration` file.
+- MUST have a documented key rotation policy.
```

Comment:
I'm not sure our existing ones have this publicly documented. This should be a MAY and maybe say "SHOULD have a plan in place for key rotation in the case of compromise" - Again, stressing we're looking for serious IDPs